pkg/subnet/local.go

```diff
@@ -47,8 +47,8 @@ type Deployer struct {
 	setDefaultSnapshot  setDefaultSnapshotFunc
 }
 
-func NewLocalDeployer(app *application.Avalanche) *Deployer {
-	return &Deployer{
+func NewLocalSubnetDeployer(app *application.Avalanche) *LocalSubnetDeployer {
```

Comment:
We decided a couple weeks ago to eliminate the duplicate `Subnet` in the name since that's the package name. I think we want to use the names `LocalDeployer` and `PublicDeployer`